guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -95,18 +75,6 @@ ifdef::katello,satellite[]
 You cannot use symbolic links for `/var/lib/pulp/`.
```

Comment:
Is this still true? As long as the correct SELinux labels are used I think the directory doesn't really matter that much anymore.